### DISCUSSION TITLE
[Tour Pauses + Breaks] Different data series for the same values

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#1052] [General]
The `true` flag is set in `pausedTimeSerie` in the time slice before a pause and in `breakTimeSerie` it is set in the time slice after a break.

I will not change it because it is already complicated.

![image](https://user-images.githubusercontent.com/1283445/229524092-0804abf9-1914-422f-83bf-d4bf28fb3865.png)

**Comment by FJBDev** [comment]
> I will not change it because it is already complicated.

It's not only because it's complicated, it's also because pauses are **very** different than breaks. 
Breaks are computed depending on [user's settings](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/preferences/general/pref-computed-values#break-time). So maybe in your example above, the breakTimeSerie is similar to the pausedTimeSerie but if you change the preferences for computing the break time, i would think it's possible that those 2 series could be different.

Also, there are some users that don't use the `breakTimeSerie` because they only want to use the `pausedTimeSerie` because it's generated based on the actions of the user on the device (I have never used computed times either because it's computed and don't represent accurately when I actually stopped/paused, i only use recorded time because I am the one that decides to do a manual pause during an activity, and I am not the only one like GurVJ said [here](https://github.com/mytourbook/mytourbook/issues/502#issuecomment-1491952977). And this is why I implemented the recorded time in [MT 20.11.0](https://mytourbook.sourceforge.io/mytourbook/index.php/history/history-2020#v20.11.0) because it's really the only relevant time for any activity

**Reply by wolfgang-ch** [reply]
> > I will not change it because it is already complicated.
> 
> It's not only because it's complicated, it's also because pauses are **very** different than breaks. 

I agree with you that break and pause times are different but setting the pause or break time flag before or after the time slice is independetly and it could be set at both locations. I cannot remember why I set it after the time slice.

The tour editor shows the break/pause flag on the 2nd time slice (0:01) but the tour chart on the 3rd time slice (0:07)

![image](https://user-images.githubusercontent.com/1283445/229564383-f392f4ac-f106-461a-ac4a-64e34fb52d59.png)
